tools/dm/manually-handling-sharding-ddl-locks.md

```diff
@@ -0,0 +1,471 @@
+---
+title: 手动处理 Sharding DDL Lock
+category: tools
+---
+
+# 手动处理 Sharding DDL Lock
+
+DM (Data Migration) 使用 sharding DDL lock 来确保各项操作以正确的顺序执行。绝大多数情况下，该锁定机制可自动完成 sharding DDL lock 同步；但在部分异常情况发生时，需要使用 `unlock-ddl-lock`/`break-ddl-lock` 手动处理异常的 DDL lock。
+
+> **注意**：
+>
+> - 不要轻易使用 `unlock-ddl-lock`/`break-ddl-lock` 命令，除非完全明确当前场景下使用这些命令可能会造成的影响，并能接受这些影响。
+> - 在手动处理异常的 DDL lock 前，请确保已经了解 DM 的分库分表合并同步原理。
```

Comment:
give a link for `分库分表合并同步原理`

Reply:
Since that file is not ready currently, I'll add it later. In fact, I'll check all DM docs add necessary links to them after all documents are ready. So you can ignore the links now. @GregoryIan